README.md

```diff
@@ -86,13 +86,19 @@ You need to install
 
 3. Copy the `.env.example` for each applications to `.env` in the same directory and fill the values required
 
-4. Start the api dev server and open `http://localhost:8000`
+4. Install new dependencies to the project
```

Comment:
Please don't add that part here. Installation here means on how to get started, that's why its under this section. 

`pnpm install` will install all required things.